pages/add/submitted.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { browserHistory, Link } from 'react-router';
+import { Helmet } from "react-helmet";
+import HintMessage from '../../components/hint-message/hint-message.jsx';
+
+const headerText = `Thanks for the submission!`;
+
+export default React.createClass({
+  getInitialState() {
+    return {
+      entryId: false
+    };
+  },
+
+  componentDidMount() {
+    let location = this.props.router.location;
+    let query = location.query;
+
+    if (query && query.entryId) {
+      let entryId = parseInt(query.entryId, 10);
+
+      // Make sure this is an entry id, and not some arbitrary data.
+      // This would be easier with != instead of !== but... linting
+      if (`${entryId}` !== query.entryId) {
```

Comment:
linting won't let me do `if (entryId != query.entryId)`, the actually *correct* way to rely on coercion